setup/base/base.go

```diff
@@ -183,13 +181,6 @@ func NewBaseDendrite(cfg *config.Dendrite, componentName string, options ...Base
 			},
 		},
 	}
-	client := http.Client{Timeout: HTTPClientTimeout}
-	if cfg.FederationAPI.Proxy.Enabled {
```

Comment:
Why are you removing this?

Reply:
Per discussion with @neilalexander above, this seems unused since a few days ago.